README.md

```diff
@@ -1,3 +1,14 @@
+# The SpatialOS CSharp Core SDK readme (EXPERIMENTAL)
```

Comment:
do we want to keep that huge warning?

Reply:
I'm so confused - I checked out master HEAD for the docs :/